pkg/controller/command/verifiable/models.go

```diff
@@ -40,9 +40,7 @@ type IDArg struct {
 // ProofOptions is model to allow the dynamic proofing options by the user.
 type ProofOptions struct {
 	// VerificationMethod is the URI of the verificationMethod used for the proof.
-	VerificationMethod string `json:"verifiableMethod,omitempty"`
-	// ProofPurpose is purpose of the proof. If omitted "assertionMethod" will be used.
-	ProofPurpose string `json:"proofPurpose,omitempty"`
+	VerificationMethod string `json:"verificationMethod,omitempty"`
 	// Created date of the proof. If omitted current system time will be used.
 	Created *time.Time `json:"created,omitempty"`
 	// Domain is operational domain of a digital proof.
```

Comment:
do we have a followup to validate that the verificationmethod can be used for the purpose?

Reply:
Now we cross check if verification method belongs to "Authentication", if not we throw error.
Added tests for the same.